README.md

```diff
@@ -1,79 +1,230 @@
+- `MongoDB Compass` We use MongoDB Compass to view our database instance.
+- `Visual Studio Code` We are in favour of using VS Code for web development for its lightweight experience and useful extensions.
+
+### Project Stack
+- `Express.js` We use Express.js to build our web server.
+- `MongoDB / Mongoose` We use MongoDB for our database, and Mongoose in order to validate data going in and out of the database.
+- `Jest + Supertest` We use Jest and Supertest as our testing libraries, to test endpoints on our Express.js server.
 
 <hr>
 
-### Running the Project
+# Contribution Guidelines
 
-1. Clone the repository to your computer.
+1. Find a 'New Ticket' you find interesting from the Project Board, and ask to be assigned.
 
-2. Open the project directory in Visual Studio Code.
+2. Once assigned, create a new branch off ```main``` named with the issue number and the ticket name formatted as follows:
 
-3. Add the ```.env``` file to the root directory. This will define our important secret keys we use for authentication, and initialization of the project.
+    - "IssueNumber-name-of-ticket"
+    -  Example: ```14-post-request-to-database-for-articles```
+    
+3. Work on your branch, commit and push when necessary. Be descriptive and clear with your commit messages.
 
-5. Run the following NPM command in the terminal at root directory. This command installs all the dependencies we use in our project. It utilizes the package-lock.json file to find which packages to install. 
+4. Once finished, create a pull request to merge into the ```main``` branch. The pull request should properly fill out the PR template.
 
-```properties
-npm i
+5. An admin will review your ticket, and either approve your changes, or request changes. If changes requested, checkout to your branch and make requested changes.
+
+***Note***: Code will be thoroughly reviewed - we uphold a code quality standard at NU Sci:
+- You will be asked to explain your reasoning to be approved.
+- You must have proper and clear formatting on all changes in the code.
+- You must not have extraneous code / code not relevant to the ticket in the pull request.
+- The code must be well-documented and commented.
+- The code must conform to the design patterns and choices already existing in the code.
+- The code must pass all tests, as well as have new tests defined for new or adjusted functionality.
+
+It often requires multiple rounds of review in order for a pull request to be merged. 
+
+<hr>
+
+# Laptop Setup + Application Installations
+- If you're new to web development, please refer to the [MacOS Setup](#macos-setup) or the [Windows Setup](#windows-setup) steps.
+- If your laptop is set up, start at the [Application Installation](#application-installation) section.
+
+## MacOS Setup
+
+### 1. Install Xcode Command Line Tools / Git
+
+- Run the following command to install Xcode Command Line Tools, which comes prepackaged with Git:
+```
+xcode-select --install
```

Comment:
just curious, is there a reason we're recommending installing via XCode rather than just `brew install git` (after installing homebrew ofc). do we use XCode for other things?

Reply:
Probably not. I just found Xcode to be generally useful in some cases. But it might not be necessary here